rskj-core/src/jmh/resources/conf/regtest.conf

```diff
@@ -60,7 +60,9 @@ trace.toAddresses=0x0000000000000000000000000000000001000008
 # eth
 ############################
 eth.blockNumber=0x15c1
-eth.transactionContractCallHash=-
+eth.transactionContractCallHash=0x8e512ffb84c535bf20f36bbcaf967eeb404e02e6a0eeaf0e490ce811d9ab0dcf
```

Comment:
This represents a Remasc Transaction, it should not be used for this benchmarking.